Podfile

```diff
@@ -135,7 +135,7 @@ target 'WordPress' do
 
     pod 'WordPressAuthenticator', '~> 1.1.8-beta.1'
     #pod 'WordPressAuthenticator', :path => '../WordPressAuthenticator-iOS'
-    #pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git' , :commit => '82900447393674532003833cc1f3a4ad1c53997f'
+    #pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git' , :commit => '895563cdd426a1b6e72702cfa40f9595ea470b74'
```

Comment:
@yaelirub  It looks like your problem may be here. You changed the commit hash but this line is actually commented out.

I think the change you actually want to make is changing above to `pod 'WordPressAuthenticator', '~> 1.1.8-beta.2'`. This should fix the conflict with lottie

Reply:
Hey @jtreanor  when I update to: pod 'WordPressAuthenticator', '~> 1.1.8-beta.2' I get:
<img width="1198" alt="screen shot 2019-01-24 at 8 09 17 am" src="https://user-images.githubusercontent.com/1335657/51691408-91f4fb00-1faf-11e9-852b-23eb5a0fb186.png">

I feel like Im in CocoaPods hell and not sure what's going on :(

Reply:
@yaelirub I _think_ if you do `pod repo update` it should be fine

Reply:
Unfortunately not in this case 😞 Using the new WordPressAuthenticator depends on this PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/10827. There were lots of updates to the dependency graph so thats why we are seeing the error.